supabase/tables/base.ts

```diff
@@ -0,0 +1,12 @@
+import { SupabaseClient } from "@supabase/supabase-js";
+import { Logs, LOG_LEVEL, LogLevel } from "@ubiquity-dao/ubiquibot-logger";
+
+export class SupaBase {
```

Comment:
```suggestion
export class Supabase {
```
Except if this names collides with something.